tests/sentry/monitors/endpoints/test_monitor_ingest_checkin_index.py

```diff
@@ -288,7 +288,7 @@ def test_with_token_auth_invalid_org(self):
         path = reverse(self.endpoint, args=[monitor.slug])
         resp = self.client.post(path, **self.token_auth_headers)
 
-        assert resp.status_code == 403
+        assert resp.status_code == 404
```

Comment:
This becomes a 404 because now the slug is different so it won't lookup an existing monitor, find it, but 403 because the org is wrong. It just won't find the monitor because the slug and org mismatch.